libs/community/pyproject.toml

```diff
@@ -49,6 +49,7 @@ requests-mock = "^1.11.0"
 langchain-core = { path = "../core", develop = true }
 langchain = { path = "../langchain", develop = true }
 langchain-standard-tests = { path = "../standard-tests", develop = true }
+mockredispy = "^2.9.3"
```

Comment:
Thank you for putting together the unit tests. We avoid adding test dependencies for specific integrations in `langchain-community` as resolving them all simultaneously can be unwieldy. We have an integration test for `RedisChatMessageHistory` [here](https://github.com/langchain-ai/langchain/blob/master/libs/community/tests/integration_tests/memory/test_redis.py); what do you think about adding to that instead?

Some instructions on running it locally for testing purposes here: https://github.com/langchain-ai/langchain/blob/2a2c0d1a9411eddfe1298f3d2bf448d192c1a6cf/libs/community/tests/integration_tests/cache/test_redis_cache.py#L22-L26